graphbolt/include/graphbolt/csc_sampling_graph.h

```diff
@@ -6,8 +6,7 @@
 #ifndef GRAPHBOLT_CSC_SAMPLING_GRAPH_H_
 #define GRAPHBOLT_CSC_SAMPLING_GRAPH_H_
 
-#include <torch/custom_class.h>
-#include <torch/torch.h>
+#include <graphbolt/sampled_subgraph.h>
```

Comment:
what's the best practice? vs `#include "sampled_subgraph.h"`

Reply:
`#include <graphbolt/sampled_subgraph.h>` is more consistent, see other files here and sparse.